ui/app/AppLayouts/Chat/views/ChatColumnView.qml

```diff
@@ -146,6 +146,8 @@ Item {
             d.restoreInputAttachments()
         }
 
+        signal updateLinkPreviewsRequested
```

Comment:
Not used. I guess it needs to replace the `updateLinkPreviews` usage

Reply:
indeed, fixed!